copy/copy.go

```diff
@@ -840,7 +840,7 @@ func (ic *imageCopier) copyLayers(ctx context.Context) error {
 		for i, srcLayer := range srcInfos {
 			err = copySemaphore.Acquire(ctx, 1)
 			if err != nil {
-				logrus.Debug("Can't acquire semaphoer", err)
+				logrus.Info("Can't acquire semaphoer", err)
```

Comment:
I know it's pre-existing, but while you're in here, can you please change "semaphoer" -> "semaphore"